content/en/tracing/setup/java.md

```diff
@@ -174,14 +174,15 @@ Don't see your desired datastores? Datadog is continually adding additional supp
 | Dropwizard Views | 0.7+     | Fully Supported | `dropwizard`, `dropwizard-view`                |
 | Hystrix          | 1.4+     | Fully Supported | `hystrix`                                      |
 | Twilio SDK       | 0+       | Fully Supported | `twilio-sdk`                                   |
+| Spring Data      | 1.8+     | Fully Supported | `spring-data`                                  |
```

Comment:
I added Spring Data to the Other section, since it isn't a Data Source per se.
I also be fine with putting in the Data Source section.